src/Draggable/Plugins/Announcement/Announcement.js

```diff
@@ -0,0 +1,162 @@
+const onInitialize = Symbol('onInitialize');
+const onDestroy = Symbol('onDestroy');
+const announceEvent = Symbol('announceEvent');
+const announceMessage = Symbol('announceMessage');
+
+const ARIA_RELEVANT = 'aria-relevant';
+const ARIA_ATOMIC = 'aria-atomic';
+const ARIA_LIVE = 'aria-live';
+const ROLE = 'role';
+
+export const defaultOptions = {
+  expire: 7000,
+  'drag:start': 'Picked up draggable element',
```

Comment:
It would be helpful to also announce the item's unique name on `drag:start` and `drag:end`, for reassurance.

Reply:
These are the fallback values, and are supposed to be customized by the user. This way, worst case, a generic message is given to a screen reader user.

Maybe we can be a little smarter and find the best alternative message, something like this maybe?:

```js
'drag:start': (event) => `Picked up ${event.source.textContent || event.source.id || 'draggable element'}`
```